config/elasticsearch.rb

```diff
@@ -0,0 +1,46 @@
+require "elasticsearch/model"
+
+# defaults
+es_client_args = { 
+  transport_options: {
+    request: {
+      timeout: 1800,
+      open_timeout: 1800,
+    }   
+  },  
+  retry_on_failure: 5,
+}
+
+# we use "production" env for all things at cloud.gov
+if ENV["RACK_ENV"] == "production"
+  vcap = ENV["VCAP_SERVICES"]
+  vcap_config = JSON.parse(vcap)
+  vcap_config.keys.each do |vcap_key|
+    if vcap_key.match(/elasticsearch/)
+      es_config = vcap_config[vcap_key]
+      es_client_args[:url] = es_config[0]["credentials"]["uri"]
+    end 
+  end 
+elsif ENV["RACK_ENV"] == "test"
```

Comment:
also, `.env.test` is not loaded till after `RACK_ENV` is evaluated. Chicken and egg to place it there.